src/ale.cpp

```diff
@@ -402,8 +397,23 @@ namespace ale {
      char *temp_str = PyBytes_AS_STRING(temp_bytes); // Borrowed pointer
      cResult = temp_str; // copy into std::string
 
-     Py_DECREF(pResultStr);
+     Py_DECREF(pResultStr); 
+     Py_DECREF(pStringFileName);
+     Py_DECREF(pStringProps);
+     Py_DECREF(pStringFormatter); 
+     Py_DECREF(pArgs);
 
      return cResult;
  }
+
+ json load(std::string filename, std::string props, std::string formatter, bool verbose) {
```

Comment:
Indentation fix

Reply:
That is indented one level as everything else because it exists in a namespace.